contracts/modules/dispute/DisputeModule.sol

```diff
@@ -239,19 +260,69 @@ contract DisputeModule is
         emit DisputeCancelled(disputeId, data);
     }
 
+    /// @notice Tags a derivative if a parent has been tagged with an infringement tag
+    /// @param parentIpId The infringing parent ipId
+    /// @param derivativeIpId The derivative ipId
+    /// @param parentDisputeId The dispute id that tagged the parent ipId as infringing
+    function tagDerivativeIfParentInfringed(
+        address parentIpId,
+        address derivativeIpId,
+        uint256 parentDisputeId
+    ) external {
+        DisputeModuleStorage storage $ = _getDisputeModuleStorage();
+
+        Dispute memory parentDispute = $.disputes[parentDisputeId];
+        if (parentDispute.targetIpId != parentIpId) revert Errors.DisputeModule__ParentIpIdMismatch();
+        if (parentDispute.currentTag == IN_DISPUTE || parentDispute.currentTag == bytes32(0))
+            revert Errors.DisputeModule__ParentNotTagged();
+
+        if (!LICENSE_REGISTRY.isParentIp(parentIpId, derivativeIpId)) revert Errors.DisputeModule__NotDerivative();
+
+        address arbitrationPolicy = $.arbitrationPolicies[derivativeIpId];
+        if (!$.isWhitelistedArbitrationPolicy[arbitrationPolicy]) arbitrationPolicy = $.baseArbitrationPolicy;
+
+        uint256 disputeId = ++$.disputeCounter;
+
+        $.disputes[disputeId] = Dispute({
+            targetIpId: derivativeIpId,
+            disputeInitiator: msg.sender,
+            arbitrationPolicy: arbitrationPolicy,
+            linkToDisputeEvidence: "",
```

Comment:
no evidence needed? or you want to leave a note here saying it's due to parent infringement?

Reply:
No evidence is needed to tag a derivative if the parent was already tagged. 

There is no `linkToDisputeEvidence` to save gas. The evidence that justified the tagging of the parent is accessible via the `parentDisputeId`.